Models/Base/JsonStat/JsonStatDatasetType.cs

```diff
@@ -2,14 +2,22 @@
 
 namespace Backend.Models.Base.JsonStat
 {
-    public class JsonStatDatasetType<T, TU> : ObjectGraphType<JsonStatDataset<T>> where T : AbstractJsonStatDimension
+    public abstract class JsonStatDatasetType<T, TU> : ObjectGraphType<JsonStatDataset<T>>
```

Comment:
With several generic types, I think it's preferable to name them `<T1, T2>`, or `<T, U>`.